demo/remLoader.js

```diff
@@ -161,25 +156,23 @@ async function createControlMeshes(controlPoints, remShaderMaterial, FlatbufferM
 
 //load REM control points
 export async function loadRemCallback(s3, bucket, name, animationEngine) {
-	let shaderMaterial = getShaderMaterial();
-	let remShaderMaterial = shaderMaterial.clone();
+	const shaderMaterial = getShaderMaterial();
+	const remShaderMaterial = shaderMaterial.clone();
 	await loadRem(s3, bucket, name, remShaderMaterial, animationEngine, (sphereMeshes) => {
-		let remLayer = new THREE.Group();
-		remLayer.name = "REM Control Points";
-		for (let ii=0, len=sphereMeshes.length; ii<len; ii++) {
-			remLayer.add(sphereMeshes[ii]);
-		}
+		const remLayer = new THREE.Group();
+    remLayer.name = "REM Control Points";
+    sphereMeshes.map(mesh => remLayer.add(mesh))
```

Comment:
Use `forEach`, rather than `map`, since you don't need to collect the values.

Reply:
Good point. I'll switch that